wiki/Help_centre/Upgrading_to_lazer/ro.md

```diff
@@ -0,0 +1,312 @@
+
+Dacă trebuie să modifici ceva la un beatmap, te rugăm să folosești editorul. În viitor vom adăuga o funcție în editor pentru a putea vedea temporar dosarul cu beatmap-uri pentru a putea fi modificat. Asta îți va permite să folosești programe externe pe un beatmap.
+
+#### Acuma că „osu!direct” este disponibil pentru toți jucătorii, ce se întâmplă cu cei care au supporter? Vor primii beneficii noi?
+
+Niște filtre în osu!direct sunt disponibile doar pentru supporteri.
+
+Pe lângă asta, suporterii mai au niște funcții deja implementate.
+
+- Supporterii pot crea playlist-uri care durează mai mult.
+- Supporterii pot activa storyboard în meniul principal.
+
+Suntem interesați să adăugăm mai multe beneficii în viitor, dar momentan lucrăm să aducem mai multe funcții din stable în lazer, așa că te rugăm să iei în considerare cumpărarea tag-ului de supporter.... ca un semn de susținere a jocului!
+
+#### Dacă folosesc coduri / cheaturi, voi primii ban?
+
+Dap.
+
+#### Dacă găsesc pe cineva care joacă cu cod / cheat pe lazer, cum ar trebuii să îl raportez?
+
+La fel cum făceai și înainte.
+
+#### Unde sunt microtranzacțiile?
+
+Cred că ai greșit jocul.
+
+## Note
+
+[^wine]: Folosind Wine.
+[^compatibility-mode]: DirectX cu modul de compatibilitate.
+[^dll]: Manual folosind fișiere `.dll`
+[^share-files]: Beatmap-urile și skin-urile vor impărții fișierele, și va salva din spațiul de stocare.
+[^gameplay-only]: Doar gameplay.
+[^online]: Prin vizualizare online.
+[^normalisation]: Aduce culorile personalizate combo din beatmap la același nivel de luminozitate.
+[^hold-for-hud]: Ține apăsat `Ctrl` pentru a vedea HUD-ul momentan atunci când este ascuns.
+[^offset-calibration-stable]: Ajustabil manual prin taste.
+[^offset-calibration-lazer]: Când reîncerci un beatmap, poți calibra offsetul bazat pe ultima ta încercare.
+[^can-disable]: Poate fi dezactivat.
+[^note-lock]: Încă există, dar n-ar trebuii să încurce.
+[^score-reset-balance]: Scorurile vor fi resetate pentru a balansa jocul.
+[^score-reset-isolated]: Scorurile vor fi resetate, momentan izolate de scorurile stable.
+[^online-content]: Acces direct la majoritatea informațiilor online.
+[^direct-supporter]: Prin osu!direct, doar osu!supporter.
+[^supporter]: Doar osu!supporter.
+[^soft-deletion]: Restabilește beatmap-uri șterse și alte date din setări. Ștergerile realizate vor fi definitiv atunci când ieși din joc.
+[^multi-room-max]: Maxim 16 jucători.
+[^map-only]: Doar beatmap-ul.
+[^all-files]: Toate fișierele.
+[^editor-precise-rotation]: Lipsa funcției precise de rotație.
+[^incompatibilities]: Niște funcții în editor vor cauza beatmap-urile să ruleze greșit în stable - va fi rezolvat în curând.
```

Comment:
```suggestion
[^incompatibilities]: Unele funcții în editor vor cauza beatmap-urile să ruleze greșit în stable - va fi rezolvat în curând.
```
Should probably be changed in the original version. (This is also no longer the case with the `.olz` and `.osz` format.) @Walavouchey would like your input on this

Reply:
(English line: https://github.com/ppy/osu-wiki/blob/master/wiki/Help_centre/Upgrading_to_lazer/en.md?plain=1#L303)